util/util.cpp

```diff
@@ -109,23 +148,32 @@ std::string getService(const std::string path)
     }
     catch (const sdbusplus::exception::SdBusError& ex)
     {
+        std::cerr << "Get dbus service fail. " << ex.what() << std::endl;
```

Comment:
Is getService() called in every loop? Looks expensive to me... Also potential log flooding?

Reply:
You're right. Good catch. I need to add a simple cache for the results of getService(), as it is extremely unlikely a sensor with the same name will change services at runtime.

Reply:
Maybe can print this log when DEBUG?

Reply:
Thanks for looking at this. This is an important error message, though, and should be printed if there's something wrong, lest the user wonder why their temperature sensor isn't showing up in the margin computation.

Reply:
I will take another look at this, though, and perhaps add some throttling to the log message. It could be annoying if printed more than once. It's important to still print it during normal execution, not DEBUG, though, because it represents a serious configuration error if getService() not found.